### ISSUE TITLE
Dynamically handle symbolization of kernel modules

### ISSUE THREAD

**Issue by rockdaboot** [open-telemetry/opentelemetry-ebpf-profiler#172]
Currently, the agent recognizes the list of kernel modules only once, at startup.

## Issue
Kernel modules can be unloaded and loaded at any time.
So they are eventually not handled correctly by the agent.

## Suggestions
Detect and load kernel modules on demand.

**Comment by rockdaboot** [comment]
Closing in favor of #211
